package-lock.json

```diff
@@ -38,6 +39,44 @@
         "node": ">=18"
       }
     },
+    "analyze-wasm": {
+      "name": "@arcjet/analyze-wasm",
+      "version": "1.0.0-alpha.34",
+      "license": "Apache-2.0",
+      "devDependencies": {
+        "@arcjet/eslint-config": "1.0.0-alpha.34",
+        "@arcjet/rollup-config": "1.0.0-alpha.34",
+        "@arcjet/tsconfig": "1.0.0-alpha.34",
+        "@bytecodealliance/jco": "1.5.0",
+        "@rollup/wasm-node": "4.28.1",
+        "@types/node": "18.18.0",
+        "expect": "29.7.0",
+        "typescript": "5.7.2"
+      },
+      "engines": {
+        "node": ">=18"
+      }
+    },
+    "analyze-wasm/node_modules/@rollup/wasm-node": {
+      "version": "4.23.0",
+      "resolved": "https://registry.npmjs.org/@rollup/wasm-node/-/wasm-node-4.23.0.tgz",
+      "integrity": "sha512-jEeXqGlX+FpWml3/A15rv1mgNqhY5dyvtX8U5w2ff0p5PkxIjU1Y/GABHEjkoE7EF29ayGBtKR6kUZ+JELtjyg==",
+      "dev": true,
+      "license": "MIT",
+      "dependencies": {
+        "@types/estree": "1.0.6"
+      },
+      "bin": {
+        "rollup": "dist/bin/rollup"
+      },
+      "engines": {
+        "node": ">=18.0.0",
+        "npm": ">=8.0.0"
+      },
+      "optionalDependencies": {
+        "fsevents": "~2.3.2"
+      }
+    },
```

Comment:
```suggestion
```
I believe this is handing around from before deps were synced?

Reply:
Yeah it was, its super annoying that it doesn't auto-remove these.